src/main/java/com/github/manolo8/darkbot/utils/login/LoginUtils.java

```diff
@@ -199,7 +218,17 @@ public static void saveCredentials(Credentials c, char[] passwd) throws GeneralS
         }
     }
 
-    public static class WrongCredentialsException extends IllegalArgumentException {
+    public static class LoginException extends  IllegalArgumentException {
+        public LoginException(){
+            this("Login Exception");
+        }
+
+        public LoginException(String s) {
+            super(s);
+        }
+    }
+
+    public static class WrongCredentialsException extends  LoginException{
```

Comment:
```suggestion
    public static class LoginException extends RuntimeException {
        public LoginException(String s) {
            super(s);
        }
    }

    public static class WrongCredentialsException extends LoginException {
```

Didn't notice wrong credentials was extending illegal arg ex, probably shouldn't it's a good time to change that